src/components/create-study-form.js

```diff
@@ -206,13 +209,13 @@ export const CreateStudyForm = () => {
 
         clearTimeout(timer.current);
         timer.current = setTimeout(() => {
-            updateStudyFormState(name);
+            updateStudyFormState(name, store.getState().user.profile.sub);
```

Comment:
use useSelector instead of accessing the store globally like this. Actually this export should be removed maybe??